LICENSE

```diff
@@ -178,15 +178,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
```

Comment:
No need to change the LICENSE file, since these are just general instructions that are actually part of the Apache license itself.

Reply:
Ok, thanks for the explanation. I will revert these changes.